### PR TITLE
Add metric of TiFlash Coprocessor

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

Add metric of tiflash coprocessor.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
need translate to English docs.
- Other reference link(s):
[tics/pull/1152: Add metrics about memory usage for cop task](https://github.com/pingcap/tics/pull/1152)
[tics/pull/1168: Add metric "number of handling cop task"](https://github.com/pingcap/tics/pull/1168)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@LittleFall Could you involve a technical review?

**Comment by windtalker** [comment]
LGTM

**Review by zanmato1984 — APPROVED** [review]
LGTM

File: tiflash/monitor-tiflash.md [hunk 1]
```diff
@@ -39,8 +39,10 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
 - Executor QPS：所有 TiFlash 实例收到的请求中，每种 dag 算子的数量，其中 table_scan 是扫表算子，selection 是过滤算子，aggregation 是聚合算子，top_n 是 TopN 算子，limit 是 limit 算子。
 - Request Duration：所有 TiFlash 实例处理 coprocessor request 总时间，总时间为接收到该 coprocessor 请求至请求应答完毕的时间。
 - Error QPS：所有 TiFlash 实例处理 coprocessor 请求的错误数量。其中 meet_lock 为读取的数据有锁，region_not_found 为 Region 不存在，epoch_not_match 为读取的 Region epoch 与本地不一致，kv_client_error 为与 TiKV 通信产生的错误，internal_error 为 TiFlash 内部系统错误，other 为其他错误。
-- Request Handle：所有 TiFlash 实例处理 coprocessor 请求处理时间，处理时间为该 coprocessor 请求开始执行到执行结束的时间。
+- Request Handle Duration：所有 TiFlash 实例处理 coprocessor 请求处理时间，处理时间为该 coprocessor 请求开始执行到执行结束的时间。
 - Response Bytes/Seconds：所有 TiFlash 实例应答总字节数。
+- Cop task memory usage: 所有 TiFlash 实例处理 coprocessor 请求占用的内存总数。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:44]:
```suggestion
- Cop task memory usage：所有 TiFlash 实例处理 coprocessor 请求占用的总内存。
```

File: tiflash/monitor-tiflash.md [hunk 2]
```diff
@@ -39,8 +39,10 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
+- Handling Request Number: 所有 TiFlash 实例正在处理的 coprocessor 请求数量之和。请求的分类与 Request QPS 中的相同.
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:45]:
```suggestion
- Handling Request Number：所有 TiFlash 实例正在处理的 coprocessor 请求数量之和。请求的分类与 Request QPS 中的相同.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**